components/event-publisher/event-output-adapters/org.wso2.carbon.event.output.adapter.ui/src/main/java/org/wso2/carbon/event/output/adapter/ui/UIEventAdapter.java

```diff
@@ -298,8 +299,9 @@ public void run() {
                         }
                     }
                 }
-            } else {
+            } else if(!isLoggedDroppedMessage) {
                 EventAdapterUtil.logAndDrop(eventAdapterConfiguration.getName(), message, "No clients registered", log, tenantId);
+                isLoggedDroppedMessage = true;
             }
```

Comment:
@keizer619 IMO, we need to improve this change. After one success operation. we need to make variable isLoggedDroppedMessage to "false"..